docs/source/geopm_agent_ffnet.7.rst

```diff
@@ -72,6 +74,35 @@ Policy Requirements
 
 The ``PERF_ENERGY_BIAS`` must be between 0 and 1.
 
+Required JSON Files
+--------------------
+
+To generate the neural network and frequency recommendation files, the following procedure
+can be followed. Note that all python files referenced can be found in the directory
+``$GEOPM_SOURCE/integration/experiment/ffnet`` unless otherwise specified.
+
+# Run the ``neural_net_sweep.py`` on microbenchmarks of interest. Some microbenchmarks
```

Comment:
Maybe link to that file's detailed description that you wrote in its README.md. If I am considering whether I want to perform these steps, then I want to understand what system state is being modified. I do like that this document keeps it brief though, because at this stage, I mostly want a high-level view.

Bonus if you can also let me understand the dimensionality of each script so I can do my own back-of-the-napkin estimate of how much system time I'm looking at using if I want to kick the tires on this feature.

Reply:
Reminder to self to add dimensionality